e2e/framework/manager/scripts/rook-dind-cluster-v1.6.sh

```diff
@@ -0,0 +1,1047 @@
+DIND_IMAGE="${DIND_IMAGE:-}"
+BUILD_KUBEADM="${BUILD_KUBEADM:-}"
+BUILD_HYPERKUBE="${BUILD_HYPERKUBE:-}"
+APISERVER_PORT=${APISERVER_PORT:-8080}
+NUM_NODES=${NUM_NODES:-2}
+LOCAL_KUBECTL_VERSION=${LOCAL_KUBECTL_VERSION:-}
+KUBECTL_DIR="${KUBECTL_DIR:-${HOME}/.kubeadm-dind-cluster}"
+DASHBOARD_URL="${DASHBOARD_URL:-https://rawgit.com/kubernetes/dashboard/bfab10151f012d1acc5dfb1979f3172e2400aa3c/src/deploy/kubernetes-dashboard.yaml}"
+E2E_REPORT_DIR="${E2E_REPORT_DIR:-}"
+
+if [[ ! ${LOCAL_KUBECTL_VERSION:-} && ${DIND_IMAGE:-} =~ :(v[0-9]+\.[0-9]+)$ ]]; then
+  LOCAL_KUBECTL_VERSION="${BASH_REMATCH[1]}"
+fi
+
+function dind::need-source {
+  if [[ ! -f cluster/kubectl.sh ]]; then
+    echo "$0 must be called from the Kubernetes repository root directory" 1>&2
+    exit 1
+  fi
+}
+
+build_tools_dir="build"
+use_k8s_source=y
+if [[ ! ${BUILD_KUBEADM} && ! ${BUILD_HYPERKUBE} ]]; then
+  use_k8s_source=
+fi
+if [[ ${use_k8s_source} ]]; then
+  dind::need-source
+  kubectl=cluster/kubectl.sh
+  if [[ ! -f ${build_tools_dir}/common.sh ]]; then
+    build_tools_dir="build-tools"
+  fi
+else
+  if [[ ! ${LOCAL_KUBECTL_VERSION:-} ]] && ! hash kubectl 2>/dev/null; then
+    echo "You need kubectl binary in your PATH to use prebuilt DIND image" 1>&2
+    exit 1
+  fi
+  kubectl=kubectl
+fi
+
+busybox_image="busybox:1.26.2"
+e2e_base_image="golang:1.7.1"
+sys_volume_args=()
+build_volume_args=()
+
+function dind::set-build-volume-args {
+  if [ ${#build_volume_args[@]} -gt 0 ]; then
+    return 0
+  fi
+  build_container_name=
+  if [ -n "${KUBEADM_DIND_LOCAL:-}" ]; then
+    build_volume_args=(-v "$PWD:/go/src/k8s.io/kubernetes")
+  else
+    build_container_name="$(KUBE_ROOT=$PWD &&
+                            . ${build_tools_dir}/common.sh &&
+                            kube::build::verify_prereqs >&2 &&
+                            echo "${KUBE_DATA_CONTAINER_NAME:-${KUBE_BUILD_DATA_CONTAINER_NAME}}")"
+    build_volume_args=(--volumes-from "${build_container_name}")
+  fi
+}
+
+function dind::volume-exists {
+  local name="$1"
+  if docker volume inspect "${name}" >& /dev/null; then
+    return 0
+  fi
+  return 1
+}
+
+function dind::create-volume {
+  local name="$1"
+  docker volume create --label mirantis.kubeadm_dind_cluster --name "${name}" >/dev/null
+}
+
+# We mount /boot and /lib/modules into the container
+# below to in case some of the workloads need them.
+# This includes virtlet, for instance. Also this may be
+# useful in future if we want DIND nodes to pass
+# preflight checks.
+# Unfortunately we can't do this when using Mac Docker
+# (unless a remote docker daemon on Linux is used)
+# NB: there's no /boot on recent Mac dockers
+function dind::prepare-sys-mounts {
+  if [[ ! ${is_moby_linux} ]]; then
+    sys_volume_args=()
+    if [[ -d /boot ]]; then
+      sys_volume_args+=(-v /boot:/boot)
+    fi
+    if [[ -d /lib/modules ]]; then
+      sys_volume_args+=(-v /lib/modules:/lib/modules)
+    fi
+    if [[ -d /dev ]]; then
```

Comment:
I dont think these mount volumes are working. You can remove them from here. They are mounted in a different place. Line 395